x/ibc/testing/chain.go

```diff
@@ -266,7 +267,8 @@ func (chain *TestChain) QueryConsensusStateProof(clientID string) ([]byte, clien
 // CONTRACT: this function must only be called after app.Commit() occurs
 func (chain *TestChain) NextBlock() {
 	// set the last header to the current header
-	chain.LastHeader = chain.CreateTMClientHeader()
+	// use nil trusted fields
+	chain.LastHeader = chain.CreateTMClientHeader(chain.ChainID, chain.CurrentHeader.Height, clienttypes.Height{}, chain.CurrentHeader.Time, chain.Vals, nil, chain.Signers)
```

Comment:
I think it would be helpful to abstract this method into a wrapper function. Say `CurrentTmHeader` or something like that